sdk/tables/Azure.Data.Tables/src/BatchItem.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Data.Tables
+{
+    /// <summary>
```

Comment:
I'm assuming the doc comments were intentionally left blank and will be filled in later.

Reply:
Correct 😄